packages/react-cards/src/components/Card/CardItem/CardItem.types.ts

```diff
@@ -12,14 +13,10 @@ export interface ICardItemSlots {
 export type ICardItemTokenReturnType = ReturnType<Extract<ICardItemComponent['tokens'], Function>>;
 export type ICardItemStylesReturnType = ReturnType<Extract<ICardItemComponent['styles'], Function>>;
 
-export interface ICardItemProps extends ICardItemSlots, IStyleableComponentProps<ICardItemProps, ICardItemTokens, ICardItemStyles> {
-  /**
-   * Defines if the default horizontal padding of the Card applies to this CardItem child or not.
-   * @defaultvalue false
-   */
-  disableChildPadding?: boolean;
-}
+export interface ICardItemProps extends ICardItemSlots, IStyleableComponentProps<ICardItemProps, ICardItemTokens, ICardItemStyles> {}
 
-export interface ICardItemTokens {}
+export interface ICardItemTokens {
+  margin?: number | string;
+}
```

Comment:
Hmm, should `margin` be specific to `CardItem` or should it actually be on `StackItem`?

Reply:
Just pushed this change.